autoload/intero/repl.vim

```diff
@@ -199,6 +206,49 @@ function! g:intero#repl#get_type_signature_line_replacement(existing_line, type_
                 \ + l:indented + [l:indent . l:suffix]
 endfunction
 
+function! intero#repl#toggle_type_on_hover() abort
+    if g:intero_type_on_hover
+        call intero#repl#disable_type_on_hover()
+    else
+        call intero#repl#enable_type_on_hover()
+    endif
+endfunction
+
+function! intero#repl#enable_type_on_hover() abort
+    let g:intero_type_on_hover = 1
+    " Make the update time shorter, so the type info will trigger faster. An alternative to
+    " lowering this is to use CursorMove/CursorMoveI.
+    set updatetime=1000
```

Comment:
This is going to override the user's configuration, regardless of what the current value is. Would be nicer to only change it if it's the default value. Or better yet, don't change it at all and document that users should change it themselves.

Reply:
Hmm, I honestly would vote for checking if it's the default and only then changing it. The type hover has little practical benefit if you have to wait 4 seconds for it to popup.

Another approach is that I switch to `CursorMove`/`CursorMoveI`, which doens't rely on update time.

Thoughts?

Reply:
I agree that we shouldn't alter that bit of the configuration, and instead make a note in the FAQ/docs.

Reply:
I'll make a note in the README as with the flag that enables it, then :)